lib/hidapi/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+export PKG_CONFIG_PATH=$BUILD_PREFIX/lib/pkgconfig
+
+echo "--------------------------"
+which pkg-config
+echo "--------------------------"
+pkg-config --debug --libs libusb-1.0
+echo "--------------------------"
+
+cmake ../ -DCMAKE_INSTALL_PREFIX="$PREFIX" -DHIDAPI_WITH_HIDRAW="FALSE" -DHIDAPI_WITH_LIBUSB="TRUE"
```

Comment:
I believe these `-D` variables are also not used.

Reply:
`HIDAPI_WITH_HIDRAW` defaults to `TRUE` which enables the hidraw backend that requires `libudev`. I tried to include it but without success. `CMAKE_INSTALL_PREFIX` seems to be also necessary to perform successful build. 
Because `HIDAPI_WITH_HIDRAW` is linux specific variable I check the OS and use correct flags for macOS and linux